### DISCUSSION TITLE
How do I use combination shortcuts in the hotkeys of M1-M4

### DISCUSSION THREAD

**Discussion by Mi3sbbbi** [seerge/g-helper#1348] [General]
How do I use combination shortcuts in the hotkeys of M1-M4

**Comment by Mi3sbbbi** [comment]
For example, to switch to a different desktop, you need to WIN+CTRL+LEFT and other key combinations to accomplish the purpose.

**Comment by seerge** [comment]
@Mi3sbbbi you can't do that directly. But you can bind M1-M4 to emit "exotic" F-keys like F23 or F24 (that are not used in real life). And then use other app (like sharpkeys or autohotkey) to bind F23 to any macro or a combination of your choice.

**Reply by Mi3sbbbi** [reply]
Is it possible for developers to integrate key combinations into applications? (I think this is a good feature, after all, CMD can't do key combinations including WIN.)

**Reply by seerge** [reply]
I don’t plan to do that, as it’s too much hassle:)

**Reply by Mi3sbbbi** [reply]
I have a mouse peripherals and I bind the WIN+CTRL+LEFT function to the side keys. Is it possible for me to bind the side key macros to M1-M4 using G-helper Settings?

**Reply by Mi3sbbbi** [reply]
Is there a corresponding hotkey code for the mouse side key

**Reply by seerge** [reply]
Try Sharpkeys or Autohotkey. Your mouse side key are already intercepted (and handled) by windows (probably as mouse button 3 / 4 / 5 or something like that). So it should be possible to do it there.

G-Helper is not a dedicated app for creating macro combinations. And I don't plan to overload it with it.

**Reply by Mi3sbbbi** [reply]
I know, I know I can make a small exe and use G-helper for this purpose, this is just a suggestion.
